.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature Request]"
+labels: 'Type: Enhancement'
+assignees: ''
+
+---
+
+Thanks so much for your interest in Dash! 
+
+Before you post an issue here, could you quickly search for your issue in the Dash [community forum](https://community.plot.ly/c/dash)? 
+
+GitHub issues are great for bug reports, the community forum is great for
+implementation questions. When in doubt, please feel free to just post the
+issue here :)
```

Comment:
Before posting an issue here, please check the Dash [community forum](https://community.plot.ly/c/dash) to see if the topic has already been discussed. The community forum is also great for implementation questions. When in doubt, please feel free to just post the issue here :)